forest/shared/src/logger/mod.rs

```diff
@@ -12,8 +12,10 @@ pub fn setup_logger(log_config: &[LogValue]) {
     logger_builder.filter(None, LevelFilter::Info);
 
     for item in log_config {
-        let level = LevelFilter::from_str(item.level.as_str())
-            .unwrap_or_else(|_| cli_error_and_die("could not parse LevelFilter enum value", 1));
+        let level = LevelFilter::from_str(item.level.as_str()).unwrap_or_else(|_| {
```

Comment:
It would be helpful to also print the module that happened to have this faulty level. A rock-solid approach would be to implement `Display` for `LogValue` and then just print the entire item. :)

Reply:
Agree.